src/state/store.js

```diff
@@ -277,17 +267,10 @@ export function fetchDataForGLAM(params) {
   );
 }
 
-function intersection(a, b) {
-  const bSet = new Set(b);
-  return new Set(
-    [...a].filter((x) => bSet.has(x)),
-  );
-}
-
```

Comment:
This is unused. It doesn't really have anything to do with this PR, but I was in the area.